actions/wait-for-deployment/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -eu
+
+while true; do
+    if kubectl get deployments -n "${NAMESPACEINUSE}" "${NAMESPACEINUSE}-jade-datarepo-ui" -o jsonpath="{.status}" | grep unavailable; then
```

Comment:
It looks like this is specific to ui. Would this be useful to be generic to use with API too? 

It might be a little more complicated with the API b/c sometimes we also need to check to make sure it's actually "up" and running with the right version. (Like [here](https://github.com/DataBiosphere/jade-data-repo/blob/develop/.github/workflows/test-runner-on-perf.yml#L123))